src/game/UIDroppedItemDlg.cpp

```diff
@@ -213,7 +213,7 @@ void CUIDroppedItemDlg::AddToItemTable(int iItemID, int iItemCount, int iOrder)
         pItemExt = CGameBase::s_pTbl_Items_Exts[pItem->byExtIndex]->Find(iItemID % 1000);
     }
     if (NULL == pItem || NULL == pItemExt) {
-        __ASSERT(0, "������ ������ ���̺� ���!!");
+        // __ASSERT(0, "������ ������ ���̺� ���!!");
         CLogWriter::Write("CUIDroppedItemDlg::AddToItemTable - Invalidate ItemID : %d", iItemID);
         return;
     }
```

Comment:
I would not comment this one out, because if the loot bundle is empty, it means that something is wrong with either how the server sends the packet or how we read it. Expected behavior would be, that there will always be at least one item in the loot bundle, which is coins or aka noah.

Reply:
I have this fixed so i will push my changes in next PR